charts/opentelemetry-kube-stack/Chart.yaml

```diff
@@ -23,7 +23,7 @@ dependencies:
     condition: crds.install
```

Comment:
can you also update the CRDs too please?

Reply:
https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1384/commits/d6c39c1e7107751d2dc581226b6ea7ac57166db4

thanks for the feedback!